docs/how_to/api.rst

```diff
@@ -0,0 +1,50 @@
+Application Programming Interface
+=================================
+
+The API is usable only if the user is authenticated or has the right permissions.
+
+For authentication, an authorization token could be sent in the headers of the request, otherwise the session authentication will be used.

```

Comment:
Is there any doc we can link here?

Reply:
Maybe this one https://github.com/openedx/edx-platform/blob/master/docs/concepts/rest_apis.rst?

Reply:
We can use this: https://docs.openedx.org/projects/edx-platform/en/latest/how-tos/use_the_api.html. Although I don't think all APIs support JWT authentication:
https://github.com/eduNEXT/eox-core/blob/1176b68864847c2d3eec412e69e4c718106c161f/eox_core/api/data/v1/viewsets.py#L39
I remember there was a list of steps for generating an API token, but I don't remember in which repo. I thought it was here.
Also, I created an issue to support JWT authentication across all APIs.

Reply:
If you don't find the instructions, I'm okay to leave these docs as is.

Reply:
We can redirect to the help for developers file that includes the [steps (8-12) ](https://github.com/eduNEXT/eox-core/blame/b784403cb3afee1a4c0718599e47e9859df92ca4/docs/help_for_devs/0001-include-test-cases-files.rst#L32-L43), what do you think?

Reply:
Oh yes! Thank you.